src/components/icons/send-arrow.tsx

```diff
@@ -0,0 +1,15 @@
+import { SVGProps } from "react";
```

Comment:
Could you kindly consider including the word **type** in the **import** statement, as mentioned in the previous comment?